FirebaseAppDistribution/Sources/FIRAppDistributionRelease.m

```diff
@@ -19,9 +19,13 @@ @interface FIRAppDistributionRelease ()
 @property(nonatomic, copy) NSString *buildVersion;
 @property(nonatomic, copy) NSString *releaseNotes;
 @property(nonatomic, strong) NSURL *downloadURL;
+@property(nonatomic) NSDate *expirationTime;
 @end
 
 @implementation FIRAppDistributionRelease
+
```

Comment:
Would it be worth making it slightly less than an hour to avoid any weird race conditions around the 1 hour mark? Maybe 59 minutes?

Reply:
Good point. I'll make it 59 minutes instead.